DQM/L1TMonitorClient/python/L1TStage2MonitorClient_cff.py

```diff
@@ -45,3 +45,7 @@
                         l1TStage2Clients
                         )
 
+# EMTF DQM quality tests
```

Comment:
This could be removed completely once the modules are added to the standard sequences.